cypress/integration/full_run_simple.spec.js

```diff
@@ -1,6 +1,5 @@
 const {
   checkTableRows,
-  checkTableRowsLite,
   getBenefitsBreakdownRowsLite,
```

Comment:
Similar to my previous comment - I think this extends to element / function naming. If we no longer have a "Lite" mode, it's a bit confusing code-wise to have the word "Lite" sprinkled through our elements and functions.